_posts/2019-11-28-jumpstart.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: JumpStart
+date: 2019-11-28
+categories:
+  - projects
+description: A display to jumpstart your day!
+image: https://www.csh.rit.edu/~mom/drno/Jumpstart.png
+image-sm: https://www.csh.rit.edu/~mom/drno/Jumpstart.png
+author: Beckett Jenen
+author-image: https://www.csh.rit.edu/~mom/drno/me.jpg
+author-bio: First year Computer Science major.
+author-email: beckettjenen@gmail.com
+author-social:
+  github: https://github.com/Dr-N0
+  linkedin: https://www.linkedin.com/in/beckettjenen/
+---
+
+# What is Jumpstart?
+Jumpstart dashboard that runs on a TV in the CSH 'Vader lobby'. Its primary goal is to display 
+relevant information to members of CSH to jumpstart their days.
+​
+# Features
+ - Displays the date & time
+ - CSH Logo that changes colors based on time of day. The logo can also changes for holidays
+ like christmas
+ - A spot for announcements. eboard members can choose to display messages posted in 
+ #announcements on Jumpstart. This was done through a custom slack app that utilizes various Slack
+ APIs.
```

Comment:
I'd re-word this bit about announcements to make less specific references to slack channels, plus some grammar fixes. Maybe:

```
 - A spot for announcements, which Eboard members can make through our Slack. This is done using a custom slack app that utilizes Slack's APIs.
```